### PR TITLE
Fix hiscores level 1 combat if unranked in combat skills

### LINKED ISSUE #2025: Hiscores level 1 combat
When a player is unranked in combat skills, have a minmum combat level of 3 to prevent it from displaying 1.

@devLotto

### PR DESCRIPTION
**Pull request by jpabeem**
Preview:

Closes #2025

### CODE REVIEW

**Comment by MagicfTail** [comment]
I know I'm nitpicking, but the lowest combat level isn't 3, but 3.4 (i think). It doesn't really make any difference, but since it's supposed to return the value without integer rounding.

**Comment by jpabeem** [comment]
Good point @MagicfTail , totally overlooked that one myself. Shall I return 3.4 instead?

**Comment by MagicfTail** [comment]
Yes, that would be more accurate

**Comment by psikoi** [comment]
Return 3.4 if virtual levels are enabled, 3 otherwise (config)

**Comment by psikoi** [comment]
Also, I think it's a matter of time before someone complains about your commit titles.
I'm no professional myself, but I found this to be very helpful: https://chris.beams.io/posts/git-commit/

**Comment by Adam-** [comment]
I think all you would have to do for this is set a lower limit on hp to 10.

File: http-api/src/main/java/net/runelite/http/api/hiscore/Skill.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public int getLevel()
 
 	public long getExperience()
 	{
-		return experience;
+		return experience < 0 ? 0 : experience;
```
Comment (Adam-) [inline http-api/src/main/java/net/runelite/http/api/hiscore/Skill.java]: This class is really just a POJO so the handling of -1 should probably not be here.

**Pull request closed without merging**